src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -172,10 +174,24 @@ public VariantContext annotateContext(final VariantContext vc,
         return variantOverlapAnnotator.annotateOverlaps(features, variantOverlapAnnotator.annotateRsID(features, annotated));
     }
 
+    /**
+     * Annotates the given variant context for called Genotypes - adds all annotations that satisfy the predicate.
+     *
+     * @see #annotateContext(VariantContext, FeatureContext, ReferenceContext, ReadLikelihoods, Predicate, Predicate)
+     */
+    public VariantContext annotateContext(final VariantContext vc,
+                                          final FeatureContext features,
+                                          final ReferenceContext ref,
+                                          final ReadLikelihoods<Allele> likelihoods,
+                                          final Predicate<VariantAnnotation> addAnnot) {
+        return annotateContext(vc, features, ref, likelihoods, addAnnot, Genotype::isCalled);
```

Comment:
The `Genotype::isCalled` is removing the annotation for no calls by default. I kept the method with the same signature to do not change the behaviour of the tools such as HC, which might prevent the change to get in soon.

If you are ok with the change for all the tools, I can remove this method and use the one with a predicate for the genotypes that does not filter anything.